### DISCUSSION TITLE
A new "create custom name"  preset after every power cycle

### DISCUSSION THREAD

**Discussion by bergie5737** [gopro/labs#1197] [Q&A]
Hi. I just noticed today that after every power cycle there is a new "Create Custom Name" preset under videos. I've done a factory reset on my Gopro twice already and re-installed Labs, but it still is happening. I have Hero13 black, and as I am new to it, there is a possibility that I've added something strange, but only used 5 x 4CC which is  BITR, EXPS, EXPT, NR01, FAST.

**Comment by dnewman-gpsw** [comment]
I not heard of this or really understand what is the error you are seeing. Can you show images please.  Try a !RESET (Labs rest not factory.)
